lib/elixir/lib/calendar.ex

```diff
@@ -67,7 +67,7 @@ defmodule Calendar do
   @doc """
   Converts the given structure into a string according to the calendar.
   """
-  @callback to_string(Date.t | NaiveDateTime.t | DateTime.t) :: String.t
+  @callback to_string(structure) :: String.t when structure: Date.t | DateTime.t | NaiveDateTime.t
```

Comment:
I think ex_doc gives names like `arg0` to the callback variables, unless they are named in the `@callback` specification.

We had this in Ecto.